cmd/daprd/components/wfbackend_actor.go

```diff
@@ -0,0 +1,24 @@
+//go:build allcomponents
```

Comment:
This should be built with stable components too, as it's built-in

```suggestion
//go:build allcomponents || stablecomponents
```